addons/sap-success-factors/install.js

```diff
@@ -0,0 +1,142 @@
+          'client/candidate/Candidate.js',
+          'client/candidate/CandidateBackground_Certificates.js',
+          'client/candidate/CandidateBackground_Education.js',
+          'client/candidate/CandidateBackground_EducationRequestBuilder.js',
+          'client/candidate/CandidateBackground_CertificatesRequestBuilder.js',
+          'client/candidate/CandidateBackground_InsideWorkExperience.js',
+          'client/candidate/CandidateBackground_InsideWorkExperienceRequestBuilder.js',
+          'client/candidate/CandidateBackground_Languages.js',
+          'client/candidate/CandidateBackground_LanguagesRequestBuilder.js',
+          'client/candidate/CandidateBackground_OutsideWorkExperience.js',
+          'client/candidate/CandidateBackground_OutsideWorkExperienceRequestBuilder.js',
+          'client/candidate/CandidateBackground_TalentPool.js',
+          'client/candidate/CandidateBackground_TalentPoolRequestBuilder.js',
+          'client/candidate/CandidateBackground_TalentPoolcorp.js',
+          'client/candidate/CandidateBackground_TalentPoolcorpRequestBuilder.js',
+          'client/candidate/CandidateComments.js',
+          'client/candidate/CandidateCommentsRequestBuilder.js',
+          'client/candidate/CandidateLight.js',
+          'client/candidate/CandidateLightRequestBuilder.js',
+          'client/candidate/CandidateProfileExtension.js',
+          'client/candidate/CandidateProfileExtensionRequestBuilder.js',
+          'client/candidate/CandidateRequestBuilder.js',
+          'client/candidate/CandidateTags.js',
+          'client/candidate/CandidateTagsRequestBuilder.js',
+          'client/candidate/JobApplicationSnapshot_Certificates.js',
+          'client/candidate/JobApplicationSnapshot_CertificatesRequestBuilder.js',
+          'client/candidate/JobApplicationSnapshot_Education.js',
+          'client/candidate/JobApplicationSnapshot_EducationRequestBuilder.js',
+          'client/candidate/JobApplicationSnapshot_InsideWorkExperience.js',
+          'client/candidate/JobApplicationSnapshot_InsideWorkExperienceRequestBuilder.js',
+          'client/candidate/JobApplicationSnapshot_Languages.js',
+          'client/candidate/JobApplicationSnapshot_LanguagesRequestBuilder.js',
+          'client/candidate/JobApplicationSnapshot_OutsideWorkExperience.js',
+          'client/candidate/JobApplicationSnapshot_OutsideWorkExperienceRequestBuilder.js',
+          'client/candidate/JobApplicationSnapshot_TalentPool.js',
+          'client/candidate/JobApplicationSnapshot_TalentPoolRequestBuilder.js',
+          'client/candidate/JobApplicationSnapshot_TalentPoolcorp.js',
+          'client/candidate/JobApplicationSnapshot_TalentPoolcorpRequestBuilder.js',
+          'client/candidate/function-imports.js',
+          'client/candidate/index.js',
+        ]),
+      },
+      metadata: {
+        fusebit: {
+          editor: {
+            navigationPanel: {
+              hideFiles: [
```

Comment:
Create one list and then filter out success-factors.js?

Reply:
Fixed